File: backend/examples/echo.js
const fs = require("fs");
let raw = process.env.API_PAYLOAD || "";
if (!raw) {
  raw = fs.readFileSync(0, "utf8") || "{}";
}
const payload = JSON.parse(raw);
const query = JSON.parse(process.env.API_QUERY || "{}");
process.stdout.write(JSON.stringify({ echo: payload, query }));

File: backend/examples/echo.py
import json
import os
import sys

raw = os.environ.get("API_PAYLOAD") or ""
if not raw:
    raw = sys.stdin.read() or "{}"
payload = json.loads(raw)
query = json.loads(os.environ.get("API_QUERY") or "{}")
print(json.dumps({"echo": payload, "query": query}, ensure_ascii=False))
